features/attached_enable.feature

```diff
@@ -7,6 +7,28 @@ Feature: Enable command behaviour when attached to an UA subscription
         Given a `xenial` machine with ubuntu-advantage-tools installed
         When I attach `contract_token` with sudo
         Then I verify that running `ua enable cc-eal` `as non-root` exits `1`
+        And I will see the following on stderr:
+            """
+            This command must be run as root (try using sudo).
+            """
+        When I run `ua enable cc-eal --beta` with sudo
+        Then I will see the following on stdout:
+            """
+            One moment, checking your subscription first
+            Updating package lists
+            (This will download more than 500MB of packages, so may take some time.)
```

Comment:
Let's see if that download will behave properly in the Jenkins node. If not we might turn this into a manual test

Reply:
Agreed

Reply:
Besides that, for the future, it will be useful to have the manual test so we can check for the Ubuntu version without messing the testing environment too much

Reply:
Tests passed fine